README.md

```diff
@@ -11,16 +14,58 @@ Source code for Foss Events India Website (https://beta.fossevents.in)
 
 Fossevents is developed on Python3(currently python3.5). First check if you have python3 installed using `which python3`. If it does not return anything, please install python3.
 
+The following setup is for fedora 22+
+
+Github setup of the repository:
+```
+Fork the repository.
+git clone git@github.com:cosmologist10/fossevents.in.git
+cd fossevents.in/  //change the directory
+git remote add upstream git@github.com:fossevents/fossevents.in.git
+Switching of branch
+```
+
 Create virtualenv using python3 executable:
 ```
 virtualenv venv -p $(which python3)
-source venv/bin/activate
+. venv/bin/activate
+```
+
+The database which we are using is Postgresql, so Postgresql installation and its setup:
+```
+sudo dnf install postgresql-server postgresql-contrib
+sudo systemctl enable postgresql
+sudo postgresql-setup --initdb   #initialise database and logs once
+sudo systemctl start postgresql
+sudo -u postgres psql
+postgres=#CREATE USER cosmologist10 WITH PASSWORD 'xxxxxx';
+postgres=#ALTER USER cosmologist10 SUPERUSER CREATEDB;
 ```
 
-Run the following command in virtual environment:
+Run the command in virtual environment, it will create database:
 ```
 createdb fossevents
+```
+
+Install the following for requirements/development.txt installation:
+```
+sudo dnf  install libxml2-dev libxslt1-dev python-dev  libpq-dev
+pip install rcssmin --install-option="--without-c-extensions"
+pip install rjsmin --install-option="--without-c-extensions"
+pip install django-compressor --upgrade
 pip install -r requirements/development.txt
+```
+
+For migration, we need to alter some trust auth in pg_hba.conf file:
```

Comment:
@aniketmaithani this is needed only the first time you setup postgresql on your machine.